libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -339,7 +339,7 @@ void YamlConfiguration::parseProvenanceReportingYaml(
     return;
   }
 
-  if (!reportNode || !(reportNode->IsSequence())) {
```

Comment:
Sorry, confused this with remote processing groups.  Regardless, another check is needed for Node typing.  In this case, we are looking for a map, or perhaps, more generically we need to just check for defined